app_test.py

```diff
@@ -79,23 +80,38 @@ def test_invite(self):
         self.assertEqual(response.status_code, 302)
 
     def test_ws_test(self):
-        """ check ws_test responds """
+        """ Check ws_test responds """
         response = self.client.get('/ws_test')
         self.assertEqual(response.status_code, 200)
 
+    def test_oauth_authorized(self):
+        """check oauth redirects """
```

Comment:
These docstrings aren't consistent with the others: `""" Check oauth redirects """`